doc/stdenv/meta.chapter.md

```diff
@@ -134,7 +134,29 @@ Attribute Set `lib.platforms` defines [various common lists](https://github.com/
 This attribute is special in that it is not actually under the `meta` attribute set but rather under the `passthru` attribute set. This is due to how `meta` attributes work, and the fact that they are supposed to contain only metadata, not derivations.
 :::
 
-An attribute set with as values tests. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise. A derivation that is a test needs to have `meta.timeout` defined.
+An attribute set with tests as values. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise. A derivation that is a test needs to have `meta.timeout` defined.
```

Comment:
```suggestion
An attribute set with tests as values. A test is a derivation, which builds successfully when the test passes, and fails to build otherwise.
```

I think I probably had added the note about `meta.timeout` when initially introducing `passthru.tests`… but it looks like the usage settled on not actually setting it, because it's cumbersome, and it totally makes sense. So maybe let's take advantage of the fixes in this line to remove this sentence, which incidentally doesn't actually match what's done in the paragraph below? :)